dotcom-rendering/src/types/mainMedia.ts

```diff
@@ -20,8 +20,9 @@ type Video = Media & {
 	image?: string;
 };
 
-type LoopVideo = Media & {
-	type: 'LoopVideo';
+type SelfHostedVideo = Media & {
+	type: 'SelfHostedVideo';
+	format: 'Default' | 'Loop' | 'Cinemagraph';
```

Comment:
`format` is a _terrible_ name as it mean something else entirely in the context of video. What's a better name? 🤔